### PR TITLE
first map draft copyedits

### PR DESCRIPTION
**Pull request by jgravois**
* added a link in the tutorials sidebar to our first actual tutorial
* used `xml` as the identifier to get <html> syntax highlighting
* trimmed down the tutorial length a bit.

what do you think about these tweaks @pmacMaps?

### CODE REVIEW

File: src/pages/tutorials/create-your-first-app.md [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+<body>  
+</body>
+</html>
+```
+
+##### Create a Container for the Map
+
+We are now ready to create a container for our map.  This will be a  `<div>`  element within the  `<body>`  element.  We must provide it with an `id` attribute.  The `id` will be passed into the code that creates the Leaflet map.  We will use an `id` of **map**.
+
+For our map to display, we must set the height and width for this element.  Since we want it to fill the whole page, we'll make sure there to remove all margins and padding too.
```
Comment (pmacMaps) [inline src/pages/tutorials/create-your-first-app.md]: we'll make sure ~~there~~ to remove all margins and padding too.

File: src/pages/tutorials/create-your-first-app.md [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+    <div id="map"></div>
+</body>
+</html>
+```
+
+##### Construct the Map Object
+
+Now its time to write some JavaScript!  This belongs inside a `<script>` tag inside the `<body>`.
+
+Our first step is to create a map object. In our constructor we must provide the `id` of the element containing the map.  Its a good idea to specify latitude and longitude coordinates for the center of the map, and the initial zoom level too.
```
Comment (pmacMaps) [inline src/pages/tutorials/create-your-first-app.md]: *It's* a good idea to specify latitude

**Comment by pmacMaps** [comment]
@jgravois I think it's great! 

I commented on a couple grammar related changes.  Other than that I think it's ready to go.

**Comment by jgravois** [comment]
awesome @pmacMaps.  thanks for the copyedits!

**Pull request merged**